securedrop/client.py

```diff
@@ -105,7 +110,6 @@ def register_new_user(self):
         else:
             raise RuntimeError("Invalid input")
 
-
```

Comment:
Remove this newline

Reply:
Gonna let the formater take care of this.